.github/workflows/ci.yml

```diff
@@ -45,6 +47,20 @@ jobs:
         shell: bash
         run: sudo make install
 
+      - name: Setup Elixir Host SDK
+        if: ${{ runner.os != 'macOS' }}
```

Comment:
This action is not supported on macOS yet. I'm testing on mac so it should be okay for now.